main.tf

```
@@ -0,0 +1,12 @@
+terraform {
+  required_version = "v0.13.2"
```

```suggestion
required_version = ">= v0.13.2"
```

Following the readme will actually install the latest version of terraform which is 0.13.3 and terraform init doesn't work for me since this line is enforcing 0.13.2

---

good call. I'll also add <0.14 just because I know we have to be careful when that jump happens